app/models/news_item.rb

```diff
@@ -19,6 +19,7 @@ class NewsItem < ApplicationRecord
   scope :tagged, -> { where(is_human_tagged: true) }
   scope :twitter, -> { where(feed_source: FeedSource.twitter) }
   scope :order_by_published, -> (order = nil) { order(feed_item_published_at: order || :desc) }
+  scope :front_page, -> { where(feed_source: FeedSource.coindesk.or(FeedSource.cointelegraph)) }
```

Comment:
Maybe worth moving this into a separate `Query` in `app/queries` since this specific to a particular view

Reply:
You're right, it probably won't be recycled. I would propose moving it to the query on the controller instead then? Setting up a new model in app/queries for it seems like it might be overkill?

Reply:
Sure that works too